hudi-common/src/main/java/org/apache/hudi/common/config/TypedProperties.java

```diff
@@ -39,7 +39,7 @@ public TypedProperties() {
     super(null);
   }
 
-  protected TypedProperties(Properties defaults) {
+  public TypedProperties(Properties defaults) {
```

Comment:
The downstream app should use `TypedProperties#copy` instead of directly using the constructor as this incurs copy operation which is expensive and should be aware of.  Thus there is no need to make the constructor public.

Reply:
This makes sense to me